include/ginkgo/core/matrix/csr.hpp

```diff
@@ -493,6 +512,12 @@ class Csr : public EnableLinOp<Csr<ValueType, IndexType>>,
         /* Use imbalance strategy when the matrix has more more than 1e8 on AMD
          * hardware */
         const index_type amd_nnz_limit{static_cast<index_type>(1e8)};
+        /* Use imbalance strategy when the maximum number of nonzero per row is
+         * more than 25600 on Intel hardware. */
+        const index_type intel_row_len_limit = 25600;
```

Comment:
small nit: These should be `static` (since all the variables should have the same value) and potentially even `constexpr` (if you don't actually need the address of these variables).
Since this is in the public interface, I assume making it `constexpr` might not be ideal, but IMO making it `static` (to have only one `const` variable, and not one per object instance) should be possible.

Reply:
I found it is in public. is it okay to change it?

Reply:
Actually, if we made this `static const`, we would have to instantiate each variable once (so only one instance is created), which is what I forgot when commenting on this issue...
I guess, for now, it is fine keeping it as it is. However, if we don't use `intel_row_len_limit` outside, it might make sense to make it `private` and `static constexpr`.